src/app/exams/exams-view.component.ts

```diff
@@ -9,13 +9,19 @@ import { CouchService } from '../shared/couchdb.service';
 import { FormControl, AbstractControl } from '@angular/forms';
 import { CustomValidators } from '../validators/custom-validators';
 import { Exam, ExamQuestion } from './exams.model';
+import { PlanetMessageService } from '../shared/planet-message.service';
 
 @Component({
   selector: 'planet-exams-view',
   templateUrl: './exams-view.component.html',
   styleUrls: [ './exams-view.scss' ]
 })
 
+// .mat-card {
```

Comment:
Make sure you remove lines like this before taking off the **WIP** label.  Code that is commented out should not be merged into the master branch.